modules/dr-restoring-cluster-state.adoc

```diff
@@ -185,6 +185,14 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
 
+. In a separate terminal window, log in to the cluster as a user with the `cluster-admin` role by using the following command:
++
+[source,terminal]
+----
+# oc login -u <cluster-admin> <1>
```

Comment:
@kalexand-rh In looking at these docs today, I noticed that this command was the only one in the procedure that used `#` as the prompt. Was that purposeful, or could this be a `$`?

Reply:
The command provided in the bug was run as root, and I don't see any other specific requirements for it.

Reply:
Thanks, I'll submit a PR to adjust it and will run it past QE again